app/controllers/posts_controller.rb

```diff
@@ -2,12 +2,12 @@ class PostsController < ApplicationController
   def index
     @user_id = params[:user_id]
     @user = User.find(@user_id)
-    @posts = @user.posts.includes(:comments)
+    @posts = @user.posts
```

Comment:
Hi @RolandM99 
When we add eager-loading there, it is pointless because we already have a function that loads the last five comments to which eager-loading has been added. We get a bullet alert that tells us to avoid eager loading here when we dot that.

Reply:
- [ ] 👍 Great job for highlighting the N+1 problem solution that you implemented [here](https://github.com/carlylechia/BLOGG_R/pull/8/files#diff-fd0046e2ea37f7ce3e91c417c57233c4b71a3324c477b5070196305ba363835cR14) in your code.